lib/utils/mixins/class_connection.dart

```diff
@@ -0,0 +1,17 @@
+class Connection {
+  late String type;
+  late bool state;
+
+  Connection(String type, bool state ){
+    this.type = type;
+    this.state = state;
+  }
+
+  String getType(){
+    return this.type;
+  }
+
+  bool getstate(){
```

Comment:
Acho que a única coisa é mudar o nome desse metodo de "getstate" para "getState" 
```suggestion
  bool getState(){
```